apps/health/tests/test_healthchecks.py

```diff
@@ -81,9 +91,12 @@ def test_health_sls_endpoint(self):
             pass
         self.assertEqual(msg, "all's well")
 
-    def test_health_sls_fail(self):
-        with HTTMock(self.fail):
-            response = self.client.get(self.url + "/health/sls")
```

Comment:
Seems like we lost the coverage for /health/sls in the new cases? Perhaps we ought to add all of the external health checks with 503 assertions here?